src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/TimeResource.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.hmppstemplatepackagename
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDateTime
+
+@RestController
+@RequestMapping("/time")
+class TimeResource {
+
+  @PreAuthorize("hasRole('TEMPLATE_EXAMPLE')")
+  @GetMapping
+  fun getTime() = "${LocalDateTime.now()}"
```

Comment:
This is the secure endpoint we were asked to add by @rj-adams. The plan is to call this from the Typescript template so it includes has an example of calling a secured endpoint. 

I like it because it's simple and there's no harm if somebody forgets to remove it - it doesn't poolute the API model. On the other hand we're going to have to add some kind of model if when we get onto the OpenAPI docs ticket as an example of an OpenAPI specification.